docs/docs/components/link.md

```diff
@@ -11,6 +11,15 @@ This component displays a hyperlink. On the web, it translates to an &lt;a&gt; t
 
 ## Props
 ``` javascript
+// Should fonts be scaled according to system setting?
+allowFontScaling: boolean = true; // Android and iOS only
+
+// Should the scale multiplier be capped when allowFontScaling is set to true?
+// The default of 0 indicates that the compoent should obey the global setting 
+// in UserInterface which by default is uncapped.
+// Note: Most versions of React Native don’t support this interface. 
```

Comment:
Rather than saying "Most versions of React Native", please change to "Older versions...".